php/tests/generated_class_test.php

```diff
@@ -877,4 +877,279 @@ public function testPrefixForReservedWords()
         $m = new \Foo\PBEmpty();
         $m = new \PrefixEmpty();
     }
+
+    #########################################################
+    # Test phpdoc for getters and setters.
+    #########################################################
+
+    public function testPhpDocForClass()
+    {
+        $class = new ReflectionClass('Foo\TestMessage');
+        $doc = $class->getDocComment();
+        $this->assertContains('Foo&#92;TestMessage', $doc);
```

Comment:
This is copied from java generated file. Java interprets Unicode escape sequences anywhere (e.g., \t will be interpreted). Will php do that even in comment? If not, we can remove it.